test/functional/configuration/configuration.spec.js

```diff
@@ -286,22 +287,54 @@ test("shows errors for duplicate names", async t => {
   await instances[1].nameField.expectError(t);
 });
 
-test("shows error for negative ID sync container ID", async t => {
+test("shows error for non-negative number for ID sync container ID", async t => {
   await extensionViewController.init(t, {});
   await instances[0].propertyIdField.typeText(t, "PR123");
   await instances[0].idSyncContainerIdField.typeText(t, "-1");
   await extensionViewController.expectIsNotValid(t);
   await instances[0].idSyncContainerIdField.expectError(t);
 });
 
-test("shows error for non-integer ID sync container ID", async t => {
+test("shows error for a float number for ID sync container ID", async t => {
   await extensionViewController.init(t, {});
   await instances[0].propertyIdField.typeText(t, "PR123");
   await instances[0].idSyncContainerIdField.typeText(t, "123.123");
   await extensionViewController.expectIsNotValid(t);
   await instances[0].idSyncContainerIdField.expectError(t);
 });
 
+test("shows error for an arbitrary string for ID sync container ID", async t => {
+  await extensionViewController.init(t, {});
+  await instances[0].propertyIdField.typeText(t, "PR123");
+  await instances[0].idSyncContainerIdField.typeText(t, "123foo");
+  await extensionViewController.expectIsNotValid(t);
+  await instances[0].idSyncContainerIdField.expectError(t);
+});
+
+test("does not show error for a data element for ID sync container ID", async t => {
+  await extensionViewController.init(t, {});
+  await instances[0].propertyIdField.typeText(t, "PR123");
+  await instances[0].idSyncContainerIdField.typeText(t, "%123foo%");
+  await extensionViewController.expectIsValid(t);
+});
```

Comment:
Do you want to test a multi-element data element?

Reply:
It's a good idea. I'll do it.